contrib/ros/util/gazebo.py

```diff
@@ -2,28 +2,18 @@
 
```

Comment:
this should be part of the Worlds package. It's hardly a util if it provides fundamental functionality.

Reply:
I have moved it to world package